### PR TITLE
Add date picker control

### PR DESCRIPTION
**Pull request by kylebshr**

### CODE REVIEW

File: DatePicker.swift [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    public func backingViewDescription(with context: ViewDescriptionContext) -> ViewDescription? {
+        NativeDatePicker.describe { config in
+            config.builder = {
+                NativeDatePicker(frame: context.bounds)
+            }
+
+            config.apply { view in
+                view.apply(model: self)
+//                print(view.frame)
```
Comment (kylebshr) [inline DatePicker.swift:70]:
```suggestion
```

File: DatePicker.swift [hunk 2]
```diff
@@ -0,0 +1,112 @@
+//
+//  DatePicker.swift
```
Comment (kyleve) [inline DatePicker.swift:2]: File's in the wrong spot

File: DatePicker.swift [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        onChange: @escaping (Date) -> Void,
+        configure: ((inout DatePicker) -> Void)? = nil
+    ) {
+        self.date = date
+        self.onChange = onChange
+        configure?(&self)
+    }
+
+    public var content: ElementContent {
+        struct Measurer {
```
Comment (kyleve) [inline DatePicker.swift:41]: Do you need this; can you just do it inline in the measure block?

**Review by kyleve — APPROVED** [review]
Changelog!

File: DatePicker.swift (hunk 2 above)
Comment (kylebshr) [reply DatePicker.swift:2]: dang it

File: DatePicker.swift (hunk 3 above)
Comment (kylebshr) [reply DatePicker.swift:41]: Need a place for the static instance, so it's nicely scoped with this type.

**Pull request closed without merging**